terraform/compute.tf

```diff
@@ -0,0 +1,21 @@
+resource "opennebula_virtual_machine" "todo-vm" {
```

Comment:
I have seen that you expect to have 2 disks for deployment.
Is`t it be easier to just make first disk larger?
If you insist on creating separate disk for app, could you please include terraform file that creates disk? I am getting a weird errors from Opennebula when trying to do so(

```
# Fetch the existing OpenNebula VM template by ID
data "opennebula_template" "vm_template" {
  id = var.template_id  # Replace with your template ID
}

# Fetch the disk image ID used in the VM template
data "opennebula_image" "template_disk_image" {
  id = data.opennebula_template.vm_template.disk[0].image_id
}

resource "opennebula_virtual_machine" "todo-vm" {

  name        = "todo-main-machine"
  description = "VM for the to-do app"
  cpu         = 2
  vcpu        = 4
  memory      = 2048
  template_id = data.opennebula_template.vm_template.id
  # group       = var.op_group
  permissions = "660"


  disk {
    image_id = data.opennebula_image.template_disk_image.id
    size     = 20000
  }

```